xscen/finalize.py

```diff
@@ -45,3 +50,125 @@ def change_units(ds: xr.Dataset, variables_and_units: dict):
                     )
 
     return ds
+
+
+def clean_up(
+    ds: xr.Dataset,
+    var_and_convert_units: Optional[dict] = None,
+    maybe_unstack_dict: Optional[dict] = None,
+    add_feb_29: bool = False,
```

Comment:
Do we want a more general `convert_calendar` instead? Do we have use cases other than re-adding Feb 29?

Reply:
ùconvert_calendar(ds_noleap, 'default')ù ne remet pas de 29 février selon mes tests... c'est pour ça que je fais ça de même. Mais je peux rajouter un ùconvert_calendarù en plus si on pense que c'est utile.

Reply:
Ah ok, je pensais que `convert_calendar` allait interpoler.

Reformulation de ma question, donc:
Est-ce qu'on veut offrir `convert_calendar` et si le calendrier final décidé par le User a plus de jours, alors dans ce cas on interpole ?

Reply:
Les dates manquantes sont ajoutées ssi `missing` est donné.

Aussi, ici, des données en '360_day' vont faire planter la fonction parce que `align_on` n'a pas été donné.

Reply:
oui, en effet missing rajoute les données manquantes. Je pense que j'étais fatiguée vendredi aprem. Les derniers changements donnent plus de flexibilité sur la conversion des calendriers.